recipe/meta.yaml

```diff
@@ -30,12 +30,15 @@ requirements:
     - pkg-config
     - llvm
     - llvmdev       # [build_platform != target_platform]
+    - jq
```

Comment:
```suggestion
```

Reply:
I added because my Mac machines at least didn't have this. I didn't check if the conda-forge containers did (and I figured it wouldn't hurt to also add to linux if that was the case).